airflow/providers/google/cloud/operators/gcs_to_gcs.py

```diff
@@ -87,6 +87,10 @@ class GCSToGCSOperator(BaseOperator):
         only if they were modified after last_modified_time.
         If tzinfo has not been set, UTC will be assumed.
     :type last_modified_time: datetime.datetime
+    :param maximum_modified_time: When specified, the objects will be copied or moved,
```

Comment:
I have a feeling that it would be much more useful to have another parameter (is_older_than) that could specify how old (or how new) objects are.

Reply:
I think that is what the maximum_modified_time is doing. It checks if objects are older than the given time and then copy it. While last_modified_time is checking if objects are new compared to the last_modified_time. Actually, that is what I think they are doing, I might be wrong though. 
When both last_modified_time and maximum_modified_time are supplied, then they act like a time interval to search for the object. I will be much happy if you can throw more light on this. Thanks for your time.

Reply:
What I meant was to add an option to specify how old the object should be - i.e. 3600 = older than 1 hour. So the thing is to specify the "age" (relative to current time) rather than modified time (absolute UTC). -> it boils down to "max_modified < current_time - age
 (so it is a simple modification) but I think it is much more useful this way  - otherwise you wil have to do this calculation in DAGs (you will rarely  pass absolute time as parameter hard-coded in DAG.).

Reply:
Ok. I understand it better now. Thanks for the explanations.

Reply:
Please verify. Thanks